pkg/controllers/managementuser/pspdelete/pspdelete.go

```diff
@@ -25,6 +29,18 @@ type handler struct {
 
 func Register(ctx context.Context, userContext *config.UserContext) {
 	starter := userContext.DeferredStart(ctx, func(ctx context.Context) error {
+		clusterName := userContext.ClusterName
+		logrus.Infof("Checking cluster [%s] compatibility before registering pspdelete controller.", clusterName)
```

Comment:
We will always log the result of the version check, So we do not gain much from telling the user we are about to check and then giving them the result in the following log statement.
```suggestion
```